src/cascadia/TerminalControl/ControlInteractivity.h

```diff
@@ -45,7 +45,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         Control::ControlCore Core();
 
         Control::InteractivityAutomationPeer OnCreateAutomationPeer();
-        ::Microsoft::Console::Types::IUiaData* GetUiaData() const;
+        ::Microsoft::Console::Render::IRenderData* GetUiaData() const;

```

Comment:
```suggestion
        ::Microsoft::Console::Render::IRenderData* GetRenderData() const;
```
Or, if we rename `IRenderData` to `ITerminalData`, we should rename it to `GetTerminalData` instead

Reply:
Same reasoning for ControlCore and HwndTerminal

Reply:
> Or, if we rename `IRenderData` to `ITerminalData`

I like that! However I'd probably do that separately at some point in the future. There's quite a lot of places which use `IRenderData` right now after all.